File: AOJ/ITP1/ITP1_10_B_Triangle.py
import math
a, b, theta = map(int, input().split())
theta = math.radians(theta)
S = a * b * math.sin(theta) / 2
L = math.sqrt(a ** 2 + b ** 2 - 2 * a * b * math.cos(theta)) + a + b
h = b * math.sin(theta)
print("{:.4f}\n{:.4f}\n{:.4f}".format(S, L, h))

File: AOJ/ITP1/ITP1_10_B_Triangle.cs
using System;
using System.Linq;

class P
{
    static void Main()
    {
        var a = Console.ReadLine().Split().Select(int.Parse).ToList();
        double r = a[2] * Math.PI / 180, s = Math.Sin(r);
        Console.WriteLine($"{a[0] * a[1] * s / 2}\n{Math.Sqrt(a[0] * a[0] + a[1] * a[1] - 2 * a[0] * a[1] * Math.Cos(r)) + a[0] + a[1]}\n{s * a[1]}");
    }
}
